src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -93,11 +101,25 @@ internal abstract partial class DefinitionItem
                 nameDisplayParts, displayIfNoReferences);
         }
 
+        // Kept around for binary compat with F#/TypeScript.
```

Comment:
> I would consider that (a source break) acceptable based on their use of internal APIs.

It is not acceptable :)   We don't break our partners, even the ones using APIs we haven't made public yet.  We can do a model of:

1. Add new APIs.
2. Insert.
3. Get other team to move over.
4. Remove.

And i'm talking with TS right now about that.  However, a source break will cause massive headaches and pain all over, and it's something we don't really ever do (intentionally).